webpack.config.js

```diff
@@ -34,6 +34,14 @@ module.exports = {
                 query: {
                     presets: ['es2015']
                 }  
+            },
+            {
+                test: /\.jsx?$/,
+                exclude: [/node_modules/, /__tests__/],
+                loader: 'babel-loader',
+                query: {
+                    presets: ['react','es2015']
+                }  
             }
         ]
     },
```

Comment:
Not sure which way is better, webpack the react stuff in bundle.js, or not to do that but reference react js scripts in html. Any thoughts on that?

Reply:
I would prefer to webpack the react stuff as it is minified and the files are further abstracted from the end users